index.js

```diff
@@ -1,21 +1,42 @@
+/*jshint node: true */
+'use strict';
+
 var path = require('path');
+var mergeTrees = require('broccoli-merge-trees');
+var Funnel = require('broccoli-funnel');
 
 module.exports = {
   name: 'ember-data-factory-guy',
 
+  treeForVendor: function() {
```

Comment:
Super .. glad you can give it a look.  I need the help. 
I am going to try again tonight, after failing for last few days over and over again. Your tip about those other repos was helpful.